<ui version="4.0" >
 <class>PastInfoBase</class>
 <widget class="QWidget" name="PastInfoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string></string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget2" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>A&amp;ffilation</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="TextLabel3" >
         <property name="text" >
          <string>Organization, Affiliation, Group:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="edtAf3" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="edtAf1" />
       </item>
       <item row="3" column="0" >
        <widget class="QComboBox" name="cmbAf3" />
       </item>
       <item row="1" column="0" >
        <widget class="QComboBox" name="cmbAf1" />
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="edtAf2" />
       </item>
       <item row="2" column="0" >
        <widget class="QComboBox" name="cmbAf2" />
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Background</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="TextLabel4" >
         <property name="text" >
          <string>Past background:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QComboBox" name="cmbBg1" />
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="edtBg2" />
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="edtBg3" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="edtBg1" />
       </item>
       <item row="2" column="0" >
        <widget class="QComboBox" name="cmbBg2" />
       </item>
       <item row="3" column="0" >
        <widget class="QComboBox" name="cmbBg3" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbAf1</tabstop>
  <tabstop>edtAf1</tabstop>
  <tabstop>cmbAf2</tabstop>
  <tabstop>edtAf2</tabstop>
  <tabstop>cmbAf3</tabstop>
  <tabstop>edtAf3</tabstop>
  <tabstop>TabWidget2</tabstop>
  <tabstop>cmbBg1</tabstop>
  <tabstop>edtBg1</tabstop>
  <tabstop>cmbBg2</tabstop>
  <tabstop>edtBg2</tabstop>
  <tabstop>cmbBg3</tabstop>
  <tabstop>edtBg3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
